sdk/boms/azure-sdk-bom/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Release History
 
+## 1.2.24 (2024-05-29)
```

Comment:
It would be good to record all new libraries added to the BOM in this version. For e.g. in this version, `azure-developer-devcenter` and `azure-ai-translation-text` are the new entries.